src/screens/SurveyDetails/index.jsx

```diff
@@ -0,0 +1,36 @@
+// This class is for projection of the component
+import React from 'react';
+
+import RatingQuestion from '../../components/RatingQuestion';
+
+export class Question extends React.Component {
+  render() {
+    const question = 'How fulfilled did you feel during this WFH period?';
+
+    return (
+      <div className="questions">
```

Comment:
1. Avoid using plural in CSS class names 

![image](https://user-images.githubusercontent.com/696529/99343307-30a6c600-28c0-11eb-97c3-693b972dd6fc.png)


2. AVoid single-word class as much as possible

![image](https://user-images.githubusercontent.com/696529/99343298-297fb800-28c0-11eb-800b-b167fe50dc70.png)

CSS conventions do apply to React as well 💡